dcpy/test/conftest.py

```diff
@@ -18,8 +27,95 @@ def aws_credentials():
 
 
 @pytest.fixture(scope="module")
-def create_bucket(aws_credentials):
+def create_buckets(aws_credentials):
     """Creates a test S3 bucket."""
     with mock_s3():
-        s3.client().create_bucket(Bucket=TEST_BUCKET_NAME)
+        for bucket in TEST_BUCKETS:
+            s3.client().create_bucket(Bucket=bucket)
         yield  ## the yield within the mock_s3() is key to persisting the mocked session
+
+
+@pytest.fixture(scope="module")
```

Comment:
Can we move this to the publishing test module?

Reply:
`mock_data_constants()` is used in both the publishing and the new packaging test modules. it used to be in `test/connectors/edm/conftest.py` but it seemed nice to delete and consolidate that file with `test/conftest.py`

`create_bucket()` already being defined at this level set the precedent, it's used in tests for connectors and utils